diff -ruN orig/cinelerra-5.1/configure.ac new/cinelerra-5.1/configure.ac
--- orig/cinelerra-5.1/configure.ac	2019-05-29 17:06:57.000000000 -0600
+++ new/cinelerra-5.1/configure.ac	2019-05-29 20:10:42.913506167 -0600
@@ -688,7 +688,7 @@
 
 CHECK_WANT([EXR], [auto], [use exr], [])
 
-CHECK_WANT([OPENCV], [no], [opencv=sys/sta/dyn,git/tar=url], [
+CHECK_WANT([OPENCV], [yes], [opencv=sys/sta/dyn,git/tar=url], [
 ])
 
 CHECK_WANT([NUMA], [auto], [system has libnuma], [
diff -ruN orig/cinelerra-5.1/plugins/Makefile new/cinelerra-5.1/plugins/Makefile
--- orig/cinelerra-5.1/plugins/Makefile	2019-05-29 17:06:57.000000000 -0600
+++ new/cinelerra-5.1/plugins/Makefile	2019-05-29 20:12:43.824654174 -0600
@@ -157,14 +157,9 @@
 	theme_suv \
 	theme_unflat \
 	theme_cakewalk \
-
-# too costly
-#	findobject \
-#	greycstoration \
-# not finished
-#	denoisemjpeg \
-# duplicate
-#	vocoder \
+	greycstoration \
+	denoisemjpeg \
+	vocoder \
 
 PLUGIN_DIR = $(BINDIR)/plugins
 DATA = $(PLUGIN_DIR)/fonts $(PLUGIN_DIR)/shapes
